src/app/components/organisms/Tabs/Tabs.sc.ts

```diff
@@ -79,6 +80,17 @@ export const TabHeader = styled.button<TabHeaderProps>`
     }
 `;
 
+export const TabHeaderText = styled.div`
+    /* stylelint-disable-next-line value-no-vendor-prefix */
+    display: -webkit-box;
+    max-height: 32px;
+    overflow: hidden;
+    line-height: 16px;
+    -webkit-line-clamp: 2;
+    /* stylelint-disable-next-line property-no-vendor-prefix */
+    -webkit-box-orient: vertical;
```

Comment:
Specific vendor prefixes do not 'feel' like a good solution.
At least not without very specific comments why.